app/views/organizations/staff/_form.html.erb

```diff
@@ -0,0 +1,70 @@
+<!-- form -->
+<%= form_with model: staff, url: "/#{current_tenant.slug}/staff", method: :post do |form| %>
```

Comment:
```suggestion
<%= form_with model: staff, url: staff_path, method: :post do |form| %>
```
I think this works as well to avoid having to hardcode the current_tenant.slug

Reply:
Hm, this gives me the error "No route matches {:action=>"show", :controller=>"organizations/staff"}, missing required keys: [:id]"

Reply:
Ahh I see now, it is `staff_index_path`, Didn't realize they do put `_index_` in there.